aspnetcore/mvc/views/working-with-forms.md

```diff
@@ -185,10 +185,9 @@ Generates the following:
 With collection properties, `asp-for="CollectionProperty[23].Member"` generates the same name as `asp-for="CollectionProperty[i].Member"` when `i` has the value `23`.
 
 
-When ASP.NET Core MVC calculates the value of `ModelExpression`, it inspects several sources, including `ModelState` and `ViewData`. Consider `<input type="text" asp-for="@Name" />`. The calculated value of `asp-for` is the first of the following that is not null:
+When ASP.NET Core MVC calculates the value of `ModelExpression`, it inspects several sources, including `ModelState` and `ViewData`. Consider `<input type="text" asp-for="@Name" />`. The calculated value attribute of `asp-for` is the first of the following that is not null:
```

Comment:
- Remove "and `ViewData`"
- `asp-for` is an *input*. So "calculated `value` attribute ~~of `asp-for`~~ is". (Note new backticks for `value`.)